client/app/bundles/momentDashboards/components/Chart.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
+import 'chart.js';
+import React from 'react';
+import { AreaChart, LineChart } from 'react-chartkick';
+
+type chartShape = {
+  xtitle: string,
+  ytitle: string,
+  data: {},
+  chartType: "Line" | "Area",
+};
+
+// ifme themed chart colors
+// TODO: auto-generate based on size of inputs if we hit > 5 categories per graph
+const colorSchemes = ['#6D0839', '#66118', '#7F503F', '#775577', '#CCAADD'];
```

Comment:
I wonder if we can reference these colours through colors.scss?

Reply:
@julianguyen It's possible via CSS Modules, with a bit of webpack configuring. https://github.com/css-modules/css-modules first example pretty much sums up the idea.